### PR TITLE
initial setup

### PR DESCRIPTION
**Pull request by patrick-mularczyk**

### CODE REVIEW

**Review by rotanimlu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+nvm install v18.12.1
+nvm use v18.12.1
+nvm alias default v18.12.1
+```
+
+Then, go into the frontend directory and run
+
+```bash
+cd frontend
+pnpm install
```
Comment (rotanimlu) [inline README.md:40]: npm

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
\ No newline at end of file
+# take-home-starter
+
+## Setup
+
+To Install
+
+- Java Version Manager: https://sdkman.io/
+- Node Version Manager: https://github.com/nvm-sh/nvm
+
```
Comment (rotanimlu) [inline README.md:9]: Mongo section is missing: 

- install Docker
- docker pull mongo
- docker run -d --name my-mongodb-container -p 27017:27017 mongo

**Review by rotanimlu — CHANGES_REQUESTED** [review]
mongo was missing rest is working fine

File: README.md (hunk 1 above)
Comment (patrick-mularczyk) [reply README.md:40]: https://pnpm.io/

File: README.md (hunk 2 above)
Comment (patrick-mularczyk) [reply README.md:17]: Done

**Review by An-Huynh-Certivity — APPROVED** [review]

**Review by rotanimlu — APPROVED** [review]

**Pull request merged**